examples/fc_example.ino

```diff
@@ -27,9 +27,6 @@
 // Support for "Numpad" mode, which is mostly just the Numpad specific LED mode
 #include "Kaleidoscope-NumPad.h"
 
-// Support for an "LED off mode"
-#include "LED-Off.h"
```

Comment:
LED-Off seems not to be supported anymore? Maybe it was renamed?

Reply:
It's included by default with LEDControl now.